mapfishapp/src/deb/resources/etc/georchestra/mapfishapp/js/wfs.servers.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Implementation detail, I've no objection on storing them into `mapfishapp` or `mapfishapp/js` if you think it is more relevant